cl/opinion_page/views.py

```diff
@@ -679,8 +742,16 @@ async def view_recap_document(
             if rd.pacer_url and redirect_or_modal:
                 redirect_to_pacer_modal = True
 
-    title = await make_rd_title(rd)
-    rd = await make_thumb_if_needed(request, rd)
+    title = make_rd_title(rd)
+    needs_thumb = rd.thumbnail_status != THUMBNAIL_STATUSES.COMPLETE
+    if all([needs_thumb, rd.has_valid_pdf, is_og_bot(request)]):
```

Comment:
So we have `is_og_bot()` but we also get passed in an `is_og_bot` bool.  Can I believe the passed in value?  And if not, should it be removed and the call be moved further up?  Right now we use both.

Reply:
Yeah, I think you're right. We can just trust what comes in and don't need to run the `is_og_bot` function.